docs/FAQ.md

```diff
@@ -137,6 +137,20 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj squash`,
+and `jj edit`. The first is the recommended default, but `jj edit` can be useful.
+When you use `jj edit`, you don't get the opportunity to review your draft before it
+is committed to the revision. This can create oversight issues when drafting over
+changes that were considered done. You should avoid using `jj edit` when the
+revision has a conflict, as the state will have plain-text conflict annotations.
```

Comment:
> as the state will have plain-text conflict annotations.

That's true for `jj new` too.

Reply:
Good point, lost sight of that. I think I've fixed that now